### PR TITLE
Remove phrase search and filtering objects

### PR DESCRIPTION
**Pull request by mckenzielee**
Filtering for SDS objects will be deprecated, and the behavior of phrase searching is changing. Removing all references to both.

### CODE REVIEW

**Comment by mckenzielee** [comment]
Need to wait for @ScottPatterson's approval.

File: Documentation/SequentialDataStore/SDS_Streams.md [hunk 1]
```diff
@@ -152,13 +152,12 @@ Returns a list of streams.
 If specifying the optional search parameter or optional filter parameter, the list of streams returned are filtered to match 
 the search/filter criteria. If neither parameter is specified, the list includes all streams 
 in the Namespace. See [Searching](xref:sdsSearching)
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Streams.md]: More 'filters' to remove.  How about: 
...If specifying the optional search query parameter, the list of streams returned will match the search criteria.  If the search query parameter is not specified, the list will include all streams in the Namespace.

File: Documentation/SequentialDataStore/SDS_Types.md [hunk 2]
```diff
@@ -1046,14 +1046,13 @@ Returns a list of types within a given namespace.
 If specifying the optional search parameter or optional filter parameter, the list of types returned are filtered to match
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Types.md]: More 'filters' to remove. How about:
...If specifying the optional search query parameter, the list of types returned will match the search criteria. If the search query parameter is not specified, the list will include all types in the Namespace.

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 3]
```diff
@@ -581,12 +581,12 @@ Returns a list of stream views within a given namespace.
 
 If specifying the optional query parameter or optional filter parameter, the list of stream views returned is filtered to match
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Views.md]: More 'filters' to remove. How about:
...If specifying the optional search query parameter, the list of stream views returned will match the search criteria. If the search query parameter is not specified, the list will include all stream views in the Namespace.

**Pull request merged**